aeron-cluster/src/test/java/io/aeron/cluster/ConsensusModuleAgentTest.java

```diff
@@ -171,7 +165,7 @@ public void shouldLimitActiveSessions()
         Tests.setField(agent, "appendPosition", mock(ReadableCounter.class));
         agent.onSessionConnect(correlationIdOne, 2, PROTOCOL_SEMANTIC_VERSION, RESPONSE_CHANNEL_ONE, new byte[0]);
 
-        clock.update(17, TimeUnit.MILLISECONDS);
+        clock.update(SCHEMA_ID, TimeUnit.MILLISECONDS);
```

Comment:
Is there are reason why we are setting the clock time to the schema id?

Reply:
accidental overuse of the number 17 :)